G1ANT.Addon.OCR.Tesseract/G1ANT.Addon.OCR.Tesseract/Commands/OcrTesseractFindCommand.md

```diff
@@ -1,40 +1,41 @@
+| `timeout`      | [timespan](G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
+| `errorcall`    | [procedure](G1ANT.Language/G1ANT.Language/Structures/ProcedureStructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](G1ANT.Language/G1ANT.Language/Structures/LabelStructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                             | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure  |
 
-For more information about `if`, `timeout`, `errorjump` and `errormessage` arguments, please visit [Common Arguments](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Common-Arguments.md)  manual page.
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
 
-This command is contained in **G1ANT.Addon.Ocr.Tesseract.dll**.
-See: [https://github.com/G1ANT-Robot/G1ANT.Addon.Ocr.Tesseract](https://github.com/G1ANT-Robot/G1ANT.Addon.Ocr.Tesseract)
+## Example
 
-**Example 1:**
-
-In this example we are opening G1ANT website firstly using `ie.open` command. Then we would like to search for word 'Robotise' on the web page. In order to do that, we can use `ocrtesseract.find`- this command searches for certain text that we should type as value for **search** argument. **area** argument enables us to choose either screen resolution or specified area where G1ANT.Robot should look for text.
+In this example the robot opens Notepad, types “*TEST*” and after a 1-second delay searches for a word “test” on the screen using OCR, then displays its coordinates in a dialog box:
 
 ```G1ANT
-ie.open url ‴g1ant.com‴
-ocrtesseract.find search ‴Robotise‴ area ‴791⫽340⫽1663⫽672‴ relative false result ♥ocrof
-timeout 4000
-dialog ♥ocrof
+program notepad
+keyboard ⋘ENTER⋙‴   TEST    ‴
+delay 1
+ocrtesseract.find search test
```

Comment:
ocrtesseract.find search test relative true

Reply:
"true" is default, so there's no need for it here

Reply:
tesseract can have real issues to find text on two screens
best practice is to use are or relative true